python/GafferUI/CompoundEditor.py

```diff
@@ -1292,6 +1292,7 @@ def __abortTabBarRearrange( self, moveEvent ) :
 		fakeReleaseEvent = QtGui.QMouseEvent(
 			QtCore.QEvent.MouseButtonRelease,
 			self.__constrainGlobalPosTo( moveEvent.globalPosition().toPoint() if Qt.__binding__ == "PySide6" else moveEvent.globalPos(), self.__qTabBar ),
+			moveEvent.globalPosition() if Qt.__binding__ == "PySide6" else moveEvent.screenPos(),
```

Comment:
I did wonder if this _TabDragBehaviour thing was responsible for the glitches we're seeing with Qt 6, but I tried  removing it and the problems remained.

Reply:
Oh, if only!